package.json

```diff
@@ -13,7 +13,7 @@
     "@blinkmobile/json-as-files": "1.1.1",
     "appdirectory": "0.1.0",
     "are-we-there-yet": "1.1.2",
-    "async": "1.5.2",
+    "async": "2.0.1",
```

Comment:
there's a heap of breaking changes in [async 2.0](https://github.com/caolan/async/blob/master/CHANGELOG.md#breaking-changes) . do any of these effect the cli ?